pkg/lib/cloudstorage/azure.go

```diff
@@ -104,8 +105,12 @@ func (s *AzureStorage) SignedURL(name string, now time.Time, duration time.Durat
 	}
 
 	parts := azblob.BlobURLParts{
-		Scheme:        serviceURL.Scheme,
-		Host:          serviceURL.Host,
+		Scheme: serviceURL.Scheme,
+		Host:   serviceURL.Host,
+		// Inject storage account to URL when testing on IP style host, eg: 127.0.0.1
+		IPEndpointStyleInfo: azblob.IPEndpointStyleInfo{
+			AccountName: s.StorageAccount,
+		},
```

Comment:
In production, I think we do not use IP style host.

Please comment this out.

Reply:
commented